tests/test_evaluate_command.py

```diff
@@ -0,0 +1,78 @@
+from unittest.mock import patch
+
+import pytest
+from click.testing import CliRunner
+from yacs.config import CfgNode
+
+from datasetinsights.commands.evaluate import cli
+from datasetinsights.estimators.factory import EstimatorFactory
+
+
+@pytest.mark.parametrize(
```

Comment:
[nit] I found it in-general harder to understand tests when using `@pytest.mark.parametrize`. This is a double edged sword. I will recommend not to use it unless you have 3+ different parameters.

Reply:
The idea is also that more can be added. What do you think makes that test hard to understand besides knowing what @pytest.mark.parametrize does?

Reply:
This is exactly what makes `@pytest.mark.parametrize` harder to understand. If the code has changed slightly, this parameterization has to be completely removed, and this test has to be broken down to a few different functions. Usually you want to have 1 input that test 1 behavior, and the other input that test slightly different behavior. So within a test function, you have exactly 1 input, a few asserts that validate the outcome of that one input.

In unittests it's usually not a big concern for some code duplication. You prefer unit of isolation (hence the name unittest)

Reply:
If the parameterization changes this would mean that there is a potential break in the cli interface. 

"Usually you want to have 1 input that test 1 behavior" This is exactly why the parametrize decorator is used. They are invoked as different test just with different arguments.